CloudFormationWindows/lansa-stack-type-win.cfn.template

```diff
@@ -276,9 +276,9 @@
         "StackDifferences": {
             "Metadata": { "Comment1": "Table of differences between the 5 Stack Types" },
             "13DBInstanceClass": { "Development": "db.t3.small", "Test" : "db.t3.large",  "Small" : "db.t3.2xlarge",  "Medium" : "db.r5.xlarge",  "Large" : "db.r5.8xlarge" },
-            "15DBEngine": { "Development": "sqlserver-web", "Test" : "sqlserver-web",  "Small" : "sqlserver-web",  "Medium" : "sqlserver-se",  "Large" : "sqlserver-ee" },
-            "19DBAllocatedStorage": { "Development": "100", "Test" : "100",  "Small" : "500",  "Medium" : "1000",  "Large" : "1000" },
-            "20DBIops": { "Development": "1000", "Test" : "1000",  "Small" : "5000",  "Medium" : "10000",  "Large" : "10000" },
+            "15DBEngine": { "Development": "sqlserver-ex", "Test" : "sqlserver-web",  "Small" : "sqlserver-web",  "Medium" : "sqlserver-se",  "Large" : "sqlserver-ee" },
+            "19DBAllocatedStorage": { "Development": "20", "Test" : "100",  "Small" : "500",  "Medium" : "1000",  "Large" : "1000" },
+            "DBIOPSRatio": { "Development": "0", "Test" : "1000",  "Small" : "5000",  "Medium" : "10000",  "Large" : "10000" },
```

Comment:
Only the Development value is correct. All the others are meant to be the default value for DBIOPS which is 10.

Reply:
I have set the default value to 10 now